scripts/report-coverage.sh

```diff
@@ -13,4 +13,5 @@ fi
 python -m coverage combine
 python -m coverage xml
 python -m coverage report -m
-bash <(curl -s https://codecov.io/bash) -Z -X gcov -X coveragepy -X search -X xcode -X gcovout -X fix -f coverage.xml
+curl -s https://codecov.io/bash -o codecov-upload.sh
```

Comment:
I'd suggest adding `-S` (show errors), `-L` (follow redirects). Maybe also `--retry 2`.

Reply:
Good idea, done (used retry 3 thought, because why not :grin:)